src/app/page.tsx

```diff
@@ -1,9 +1,154 @@
-import styles from './page.module.css';
+'use client';
+
+import COLORS from '@/styles/colors';
+import Image from 'next/image';
+import { H2, H3, H4 } from '../styles/text';
+import { LinkButton } from '../components/Button';
+import homepageImage from '../../public/images/HomePageImage.webp';
+import * as Styles from './styles';
+
+type Stat = {
+  number: string;
+  label: string;
+};
+
+const stats: Stat[] = [
+  { number: '181+', label: 'Matched Cases' },
+  { number: '181+', label: 'Attorneys' },
+  { number: '65+', label: 'Interpreters & Translators' },
+  { number: '46+', label: 'Successful Outcomes' },
+];
```

Comment:
@varortz @kyrenetam what's the plan for the stats?
are we going for 180+, 180+, 65+, 45+ or 181, 181, 66, 46?

Reply:
plus signs! We're going with this approach given that the stats we have are based on data from their airtable (2020 onwards), and they mentioned they had more data somewhere else so we should maybe account for that missing data with the `+` signs

Reply:
should we use 180+, 180+, 65+, and 45+?
for some reason a non-multiple of 5 looks weird

Reply:
I don't really have a strong opinion on this. I see your point, but I also feel like it can make the stats feel more legit lol. @kyrenetam @kevin3656 what do y'all think? I'm ok with either approach.

Reply:
Oh wait (I forgot to bring this up during worksesh), Ambreen also asked if we could add functionality so that admin can update these stats through Retool. Definitely something we should implement if we have time!